splunkapp/generate_wazuh_splunk_app.sh

```diff
@@ -101,47 +108,45 @@ main() {
                 ;;
             "-s"|"--store")
                 if [ -n "$2" ]; then
-                    if [[ "${2: -1}" != "/" ]]; then
-                    DESTINATION="$2/"
-                    else
-                    DESTINATION="$2"
-                    fi
-                    HAVE_DESTINATION=true
+                    OUTDIR="$2"
```

Comment:
Care with final slashes.

Reply:
We will take this into consideration in the future.